utils/softhsm/softhsm.go

```diff
@@ -0,0 +1,63 @@
+/*
+   Copyright The ocicrypt Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package softhsm
+
+import (
+	"bytes"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+/* RunSoftHSMSetup runs 'softhsm_setup setup' and returns the public key that was displayed */
+func RunSoftHSMSetup(softhsmSetup string) (string, error) {
```

Comment:
i was hoping there would be a good way to encapsulate this between the variable in the test file, and this package.. if this is only used for the test, could we set an ENV variable. And add a check for the env variable, if it doesn't exist, then skip the tests. (since it could be triggered by someone else using the package.

Reply:
No sure what you mean with 'variable in the test file'.

Reply:
this is a copy of the other comment i had, im going to resolve this one.